src/pretix/presale/templates/pretixpresale/event/voucher.html

```diff
@@ -410,7 +410,16 @@ <h4 id="item-{{ item.pk }}-legend">{{ item.name }}</h4>
             {% eventsignal event "pretix.presale.signals.voucher_redeem_info" voucher=voucher %}
             {% if event.presale_is_running and options > 0 %}
                 <div class="row checkout-button-row">
-                    <div class="col-md-4 col-md-offset-8 col-xs-12">
+                    <div class="col-md-4 col-md-offset-8 col-xs-12 text-center">
+                        {% if voucher.min_usages > 1 %}
+                            <p class="text-muted">
+                                <small>
+                                    {% blocktrans trimmed with number=voucher.min_usages %}
+                                        You need to select at least {{ number }} products.
+                                    {% endblocktrans %}
```

Comment:
This is a little bit untrue, as if the voucher has already been redeemed enough time, the min_usages do not apply anymore.

Reply:
thanks, fixed in 248e22c3d